python/databases/inversekinematics.py

```diff
@@ -143,10 +143,8 @@
 __copyright__ = 'Copyright (C) 2009-2012 Rosen Diankov <rosen.diankov@gmail.com>'
 __license__ = 'Apache License, Version 2.0'
 
-if not __openravepy_build_doc__:
-    from numpy import *
-else:
-    from numpy import array
+from numpy import array, float64, double
+from numpy import pi, sqrt, eye, cross, dot, linalg, reshape, mean, median, mod, random, sum, abs, all
```

Comment:
Why are these separate?

Reply:
The background is I feared long lines.

There could be three areas: modules, types, and functions/variables.

I still clearly separate lines for openrave modules import (in test/).

However, separating types (eg array/float64) and functions lines does not seem practical.

For this, maybe I can make long lines.